app/src/ui/changes/filter-changes-list.tsx

```diff
@@ -349,7 +370,11 @@ export class FilterChangesList extends React.Component<
 
   private onIncludeAllChanged = (event: React.FormEvent<HTMLInputElement>) => {
     const include = event.currentTarget.checked
-    this.props.onSelectAll(include)
+
+    const filteredItemPaths = [...this.state.filteredItems.values()].map(
+      i => i.change.path
+    )
+    filteredItemPaths.map(path => this.props.onIncludeChanged(path, include))
```

Comment:
Doesn't change the behavior, but maybe it makes more sense to use `each`?

```suggestion
    filteredItemPaths.each(path => this.props.onIncludeChanged(path, include))
```